src/cli/one_helper/oneimage_helper.rb

```diff
@@ -348,18 +356,36 @@ def format_resource(image, _options = {})
         CLIHelper.print_header(str_h1 % 'IMAGE TEMPLATE', false)
         puts image.template_str
 
-        puts
-        CLIHelper.print_header('VIRTUAL MACHINES', false)
-        puts
-
         vms=image.retrieve_elements('VMS/ID')
 
         return unless vms
 
-        vms.map! {|e| e.to_i }
-        onevm_helper=OneVMHelper.new
-        onevm_helper.client=@client
-        onevm_helper.list_pool({ :ids=>vms, :no_pager => true }, false)
+        if image.type_str.casecmp('backup').zero?
+            CLIHelper.print_header(str_h1 % 'BACKUP INFORMATION', false)
+            puts format(str, 'VM', vms[0])
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against an allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html